src/dfx/assets/project_templates/svelte/src/__frontend_name__/svelte.config.js

```diff
@@ -11,9 +11,27 @@ const config = {
       assets: 'dist',
       fallback: undefined,
       precompress: false,
-      strict: true,
+      strict: true
     }),
-  },
+    csp: {
```

Comment:
A comment would be nice here. Maybe something like:

```suggestion
    // We set the Content-Security-Policy header here because Svelte adds an hash for all the generated inline styles and scripts (see https://svelte.dev/docs/kit/configuration#csp).
    // Therefore, we must not set the Content-Security-Policy header in the `static/.ic-assets.json5` file to avoid overriding the header.
    csp: {
```